Modules/user/rememberme_model.php

```diff
@@ -40,16 +40,29 @@ class Rememberme {
     // ---------------------------------------------------------------------------------------------------------
     public function __construct($mysqli)
     {
-            $this->mysqli = $mysqli;
-            $this->log = new EmonLogger(__FILE__);
+        $this->mysqli = $mysqli;
+        $this->log = new EmonLogger(__FILE__);
+        
+        if (is_https()) {
+            $this->secure = true;
+        }
     }
 
     // ---------------------------------------------------------------------------------------------------------
     public function setCookie($content,$expire) 
     {
         $this->log->info("setCookie: $content $expire");
         
-        setcookie($this->cookieName,$content,$expire,$this->path,$this->domain,$this->secure,$this->httpOnly);
+        // setcookie($this->cookieName,$content,$expire,$this->path,$this->domain,$this->secure,$this->httpOnly);
+        // May be limited to PHP7.3
+        setcookie($this->cookieName,$content, [
+            'expires' => $expire,
+            'path' => $this->path,
+            'domain' => $this->domain,
+            'secure' => $this->secure,
+            'httponly' => $this->httpOnly,
+            'samesite' => 'Strict'
```

Comment:
Could also be defined like this https://stackoverflow.com/a/46971326/50913

Which could be a way to make it work for lower versions